scripts/cache_filter_clean.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+temp_out=$(mktemp)
+mv $temp_out $temp_out.ipynb
+cat > $temp_out.ipynb
+jupyter nbconvert --to notebook --config scripts/convert_config.py $temp_out.ipynb --output $temp_out.ipynb 1>&2
+jupyter nbconvert --to notebook --ClearOutputPreprocessor.enabled=True $temp_out.ipynb --output $temp_out.ipynb 1>&2
```

Comment:
can use `--stdout`, since this is the last step in the pipeline (doesn't work with smudge filter, since there we need to do two things).